packages/dds/map/src/test/directory.spec.ts

```diff
@@ -101,21 +101,6 @@ describe("Directory", () => {
                 assert.equal(serialized, "{}");
             });
 
-            it("Should serialize a directory without subdirectories as a JSON object", () => {
```

Comment:
why is this deleted?

Reply:
It got deleted by accident when I was deleting the tests that I added to test references out of a DDS.